sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -181,29 +181,32 @@ case class CreateViewCommand(
    * Permanent views are not allowed to reference temp objects, including temp function and views
    */
   private def verifyTemporaryObjectsNotExists(sparkSession: SparkSession): Unit = {
```

Comment:
for the sake of consistency, I think it'd be better to use either `sparkSession` or `session` like the below methods after this one